src/ipips/ipip-0499.md

```diff
@@ -0,0 +1,102 @@
+---
+# IPIP number should  match its pull request number. After you open a PR,
+# please update title and update the filename to `ipip0000`.
+title: "IPIP-0499: CID Profiles"
+date: 2025-04-03
+ipip: proposal
+editors:
+  - name: Michelle Lee
+relatedIssues:
+  - n/a
+order: 0000
+tags: ['ipips']
+---
+
+## Summary
+
+<!--One paragraph explanation of the IPIP.-->
+This proposal introduces profiles for IPFS CIDs. Profiles explicitly define CID version, hash algorithm, chunk size, DAG width, layout, and other parameters. 
+
+## Motivation
+
+Currently, CIDs can be generated with a variety of settings and optimizations for chunking, DAG width, and more. This means the same file can yield multiple, different CIDs depending on which tools and settings are used, and it is not possible to reliably reproduce or verify the CID. Profiles offer With profiles, following the same profile will produce identical CIDs for identical content, whic makes verification regardless of implementation.
+
+## Detailed design
+
+We introduce a profile naming system, 
+
+Each profile must specify the following characteristics:
+
+1. CID version (CIDv0 or CIDv1)
+2. Hash algorithm
+3. Chunk size
+4. DAG width
+5. DAG layout
+6. Required
+
+Additional profiles can be added at a future date. Profile names may be chosen from the names of any botanical tree with compound leaves.
+
+|             | Helia default | Kubo default                | Storacha default | "test-cid-v1" profile | DASL          |
```

Comment:
This is only one of Kubo presets – basically "CIDv0" as CIDv1 + raw leaves. 
https://github.com/ipfs/kubo/pull/10774 renames `test-cid-v1` to `legacy-cid-v1` to make it more clear (I'll merge this comment once it lands)

```suggestion
|             | Helia default | Kubo default                | Storacha default | Kubo `legacy-cid-v1` profile | DASL          |
```